filebeat/module/apache2/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- key: apache2
+  title: "apache2"
```

Comment:
We should probably capitalise all titles. We could do this also in the script. @dedemorton did for this some cleanup in Metricbeat.